PokeAlarm/Manager.py

```diff
@@ -679,23 +679,26 @@ def process_pokestop(self, stop):
             thread.join()
 
     def process_gym(self, gym):
+        gym_id = gym['id']
```

Comment:
I would put the `if self.__gym_settings['enabled'] is False:` check at the very beginning and remove the `if self.__gym_settings['enabled'] is True:` later on.